geode-core/src/main/java/org/apache/geode/internal/cache/execute/FunctionContextImpl.java

```diff
@@ -44,19 +45,22 @@
   private ResultSender resultSender = null;
 
   private final boolean isPossDup;
+  private DistributedSystem distributedSystem;
 
   public FunctionContextImpl(final Cache cache, final String functionId, final Object args,
-      ResultSender resultSender) {
-    this(cache, functionId, args, resultSender, false);
+      ResultSender resultSender, final DistributedSystem distributedSystem) {
```

Comment:
I don't think we need to pass the DS through. The cache already contains the DS.